charts/performancetesting/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: performancetesting
+description: Chart for saving GatlingJarUrbanOs jar
```

Comment:
Can we change this to describe the functionality in relation to the system a bit more clearly?

Something like "Cronjob for executing the system test image on cadence"